src/libs/RandomAvatarUtils.ts

```diff
@@ -0,0 +1,99 @@
+import type {FC} from 'react';
+import type {SvgProps} from 'react-native-svg';
+import Avatar1 from '@assets/images/avatars/user/default-avatar_1.svg';
```

Comment:
It's probably a good idea to use a dynamic `import` to lazy-load the SVGs only as they're needed.